api/proto/config/cipherinfo.proto

```diff
@@ -0,0 +1,61 @@
+// Copyright(c) 2017-2018 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+option go_package  = "github.com/lf-edge/eve/api/go/config";
+option java_package = "com.zededa.cloud.uservice.proto";
+
+import "devcommon.proto";
+
+// Cipher information for sensitive data.
+
+// Security Key Exchange Method
+enum KeyExchangeScheme {
+    KEA_NONE = 0;
+    KEA_ECDH = 1;
+}
+
+// Encryption Scheme for Cipher Payload
+enum EncryptionScheme {
+    SA_NONE = 0;
+    SA_AES_256_CFB = 1;
+}
+
+// Cipher information to decrypt Sensitive Data
+message CipherContext {
+    UUIDandVersion uuidandversion = 100;
```

Comment:
UUID for what? The deviceUUID? That doesn't have a version. And you have the deviceCertId below.

Reply:
I was thinking, in terms of certificate management, in the context of cloud/EVE, to put a ID field for better tracking.

Reply:
We use a truncated hash of the certificate for that. See senderCertHash and hashAlgorithm in api/proto/auth//auth.proto

I think it makes sense to use the same things for object security and not assume a UUID.